packages/react/src/components/Storage/FileUploader/Previewer/index.tsx

```diff
@@ -29,7 +29,15 @@ export function Previewer({
   isSuccess,
   percentage,
   onFileClick,
+  isEditingNames,
+  saveEdit,
+  cancelEdit,
+  startEdit,
 }: PreviewerProps): JSX.Element {
+  const uploadedFiles = () => files.filter((_, i) => fileStatuses[i]?.success);
+
+  const remainingFiles = () =>
+    files.filter((_, i) => !fileStatuses[i]?.success);
```

Comment:
Do we really need both of these to derive the values used below?
```ts
const uploadedFilesLength = () => files.filter((_, i) => fileStatuses[i]?.success).length;
const remainingFilesLength = files.length - uploadedFilesLength
```

Reply:
I'll update it